topics/proteomics/tutorials/biomarker_selection/tutorial.md

```diff
@@ -0,0 +1,376 @@
+* "tissue enriched" (expression in one tissue at least fivefold higher than all other tissues),
+* "group enriched" (fivefold higher average TPM in a group of two to to seven tissues compared to all other tissues) and
+* "tissue enhanced" (fivefold higher average TPM in one or more tissues/cell lines compared to the mean TPM for all tissues)
+
+This information is listed in the column 4 : "RNA tissue category" of the result dataset.
+
+Let's use the "Filter by keywords and/or numerical value" tool to select the candidate biomarkers based on this
+"RNA tissue category" criterium.
+
+
+> ### {% icon hands_on %} Hands-on: Filter by keywords and/or numerical value
+>
+> 1. **Filter by keywords and/or numerical value** {% icon tool %} with the following parameters:
+>    - In *"Filter by keywords"*:
+>        - {% icon param-repeat %} *"Insert Filter by keywords"*
+>            - *"Column number on which to apply the filter"*: `c4`
+>            - *"Enter keywords"*: `copy/paste`
+>                - *"Copy/paste keywords to find (keep or discard)"*: `enriched enhanced`
+>    - *"Sort by column ?"*: `Yes`
+>
+>   > ### Output
+>   > - **Filtered Add_expression_data_on_data_8**: output list of the heart biomarkers with RNA tissue category
+>   > containing "enriched" or "enhanced" (115 lines = what we are interested in)
+>   > - **Filtered Add_expression_data_on_data_8 - discarded lines**: output list of the heart biomarkers with
+>   > RNA tissue category NOT containing "enriched" or "enhanced" (not what we are interested in)
+>   {: .comment}
+{: .hands_on}
+
+We now have identified 115 candidates considered to have significantly higher expression in heart muscle according to HPA criteria.
+Let's call the dataset where are those 115 candidates '**heart115**'.
+
+
+# Annotation of this protein list with biochemical and cellular features
+
+
+Candidate biomarkers we want to identify have to be cytoplasmic and without transmembrane domains (TMD).
+Thus we will retrieve protein features from neXtProt to retrieve those informations.
+
+Since HPA only considers ENSG identifiers (related to the gene), although neXtProt uses UniProt
+identifiers (related to proteins), first thing to do is to map the Ensembl identifiers contained our list of (115)
+candidates to their corresponding UniProt accession number. The tool **ID Converter** is what we need to do so.
+
+
+> ### {% icon hands_on %} Hands-on: ID Converter
+>
+> 1. **ID Converter** {% icon tool %} with the following parameters:
+>    - *"Enter IDs"*: `Input file containing IDs`
+>    - *"Species"*: `Human (Homo sapiens)`
+>        - *"Type/source of IDs"*: `Ensembl gene ID (e.g. ENSG00000166913)`
+>        - In *"Target type"*:
+>            - *"Target type of IDs you would like to map to"*: `Uniprot accession number, Uniprot ID`
+>
+>   > ### Output
+>   > **ID converter on data 11**: In this dataset, 2 columns (columns 6 and 7, at the end) which contain
```

Comment:
I like that you discuss the outputs! I would not include the "on data 11" parts because this might not be the same for everybody (for example if the do some wrong steps somewhere in between)

also, in addition to discussing the format (what each column is) maybe can also discuss the what the outputs tell us about the samples on a biological level? What do you think?